packages/@react-spectrum/slider/docs/Slider.mdx

```diff
@@ -0,0 +1,169 @@
+the Slider's minimum and maximum values, each of which default to 0 and 100 respectively.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState(25);
+  return (
+    <Flex gap="size-150">
+      <Slider
+        label="Cookies to buy (Uncontrolled)"
+        defaultValue={25} />
+      <Slider
+        label="Cookies to buy (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue` props.
+
+```tsx example
+<Slider
+  label="Cookies to buy"
+  minValue={50}
+  maxValue={150}
+  defaultValue={100} />
+```
+
+The slider value can be formatted by using the `formatOptions` prop.
+`formatOptions` is compatible with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) and is applied based on the current locale.
+
+```tsx example
+<Slider
+  label="Currency"
+  formatOptions={{style: 'currency', currency: 'JPY'}}
+  defaultValue={60} />
+```
+
+## Labeling
+Value labels are shown above the Slider by default, but they can also be moved to the side or hidden as needed.
+The label text should be in sentence case.
+
+```tsx example
+<Flex direction="column" maxWidth="size-3000" gap="size-300">
+  <Slider label="Cookies to buy" defaultValue={25} />
+  <Slider label="Donuts to buy" labelPosition="side" defaultValue={25} />
+  <Slider label="Pastries to buy" showValueLabel={false} defaultValue={25} />
+</Flex>
+```
+
+The value label can be updated via the `valueLabel` prop in cases where showing a different scale makes sense. The `valueLabel`
+should be appropriately translated.
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState(25);
+  return <Slider label="Cookies to buy" marginBottom="size-300" value={value} onChange={setValue} valueLabel={`${value} of 100`} />
+}
+```
+
+By default, the value label is formatted as a plain number. This can be customized using the following props.
+
+* `showValueLabel`: Allows you to display or hide the value label.
+* `formatOptions`: Allows you to customize the format of the value.
+* `valueLabel`: Allows you to customize the value label to any string.
+
+```tsx example
+function Example() {
+  let [donuts, setDonuts] = React.useState(54);
+  return (
+    <Flex direction="column" maxWidth="size-3000" gap="size-300">
+      <Slider
+        label="Cookies to buy"
+        showValueLabel={false}
+        defaultValue={90} />
+
+      <Slider
+        label="Donuts to buy"
+        maxValue={60}
+        valueLabel={`${donuts} of 60 Donuts`}
+        value={donuts}
+        onChange={setDonuts} />
+
+      <Slider
+        label="Percent donus eaten"
+        maxValue={1}
+        formatOptions={{style: 'percent', minimumFractionDigits: 2}}
+        defaultValue={0.8912} />
```

Comment:
I just noticed that this Slider can only be dragged to 0 or 1 respectively, does it need a `step` defined?

Reply:
yep! good catch, thanks